### ISSUE TITLE
Couldn't Edit dates Manually on DateRangePicker

### ISSUE THREAD

**Issue by kaushik1094** [airbnb/react-dates#1333]
I am on react-dates version:  17.2.0

I am unable to edit dates manually in DateRangePicker. 

My DateRangePicker looks like this:
<DateRangePicker
                                    startDate={this.state.dateRange[0]} // momentPropTypes.momentObj or null,
                                    startDateId="your_unique_start_date_id" // PropTypes.string.isRequired,
                                    endDate={this.state.dateRange[1]} // momentPropTypes.momentObj or null,
                                    endDateId="your_unique_end_date_id" // PropTypes.string.isRequired,
                                    onDatesChange={this.onDateRangeChanged} // PropTypes.func.isRequired,
                                    focusedInput={this.state.focusedDate}  // PropTypes.oneOf([START_DATE, END_DATE]) or null,
                                    onFocusChange={this.onDateRangeFocus} // PropTypes.func.isRequired,
                                    isOutsideRange = {() => false} 
                                    orientation = {VERTICAL_ORIENTATION}                          
                                />
 I was watching an Issue where one of the Contributor mentioned to create a new issue if this happens in latest version: https://github.com/airbnb/react-dates/issues/79#issuecomment-375918552
Is this issue fixed. If so, How can I enable editing? as it is right now, I cannot edit the date range manually.

It is performing as it is showed in the storybook

**Comment by majapw** [comment]
@kaushik1094 can you share a gif of the behavior you are seeing? Are you able to select a date by clicking on the calendar?

Can you also share what browser you are using?

**Comment by kaushik1094** [comment]
@majapw Yes I am able to select the date in the Calendar. But couldn't edit the input fields of the start date and end date manually. I am using Google Chrome .

**Comment by kaushik1094** [comment]
Is there any solution for this issue yet? I am kind of wondering to add logic in the component I am using it work dynamically instead if there is any update I need to make that would be great as well :)

**Comment by Iulia-Soimaru** [comment]
I have the same issue, can't manually update the input.

When I click on the input I can see the cursor, and I can use arrows left and right to move the cursor, but deleting value or typing something in doesn't work, and it doesn't throw an error :(

**Comment by kaushik1094** [comment]
@Iulia-Soimaru What version of react-dates are you using. May be updating it to the latest version might help. But, I think this is still a bug which needs to be addressed. @majapw

**Comment by kaushik1094** [comment]
@Iulia-Soimaru try to update to the latest version 18.3.1(Don't include ^) and see if it reproduces the same error.

**Comment by ljharb** [comment]
Definitely always include `^`.

**Comment by majapw** [comment]
I wonder if v16 had the code that automatically disabled typing in the vertical orientation. I *think* it's likely fixed in the latest.

**Comment by kaushik1094** [comment]
> I wonder if v16 had the code that automatically disabled typing in the vertical orientation. I _think_ it's likely fixed in the latest.

@Iulia-Soimaru That might be the case. try to update to the latest version and see if that works
